experiment_prototype/scan_classes/sequences.py

```diff
@@ -48,54 +51,25 @@ class Sequence(ScanClassBase):
     seqtime
-        the amount of time for the whole sequence to transmit, until the logic signal 
+        the amount of time for the whole sequence to transmit, until the logic signal
         switches low on the last pulse in the sequence (us).
-    sstime  
+    sstime
         ssdelay + seqtime (total time for receiving) (us).
     numberofreceivesamples
-        the number of receive samples to take, given the rx rate, during 
+        the number of receive samples to take, given the rx rate, during
         the sstime.
     first_rx_sample_time
         The location of the first sample for the RX data, in time, from the start of the TX data.
         This will be calculated as the time at centre sample of the first pulse. In seconds.
     blanks
         A list of sample indices that should not be used for acfs because they were samples
         taken when transmitting.
-    
-    **Pulses is a list of pulse dictionaries. The pulse dictionary keys are:**
-    
-    isarepeat
-        Boolean, True if the pulse is exactly the same as the last pulse in the sequence.
-    pulse_timing_us
-        The time past the start of sequence for this pulse to start at (us).
-    slice_id
-        The slice_id that corresponds to this pulse and gives the information about the 
-        experiment and pulse information (frequency, num_ranges, first_range, etc.).
-    slice_pulse_index
-        The index of the pulse in its own slice's sequence. 
-    pulse_len
-        The length of the pulse (us)
-    intra_pulse_start_time
-        If the pulse is combined with another pulse and they transmit in a single USRP 
-        burst, then we need to know if there is an offset from one pulse's samples being
-        sent and the other pulse's samples being sent. 
-    combined_pulse_index
-        The combined_pulse_index is the index corresponding with actual number of pulses 
-        that will be sent to driver, after combinations are completed. Multiple pulse 
-        dictionaries in self.pulses can have the same combined_pulse_index if they are 
-        combined together, ie are close enough in timing that T/R will not go low 
-        between them, and we will combine the samples of both pulses into one set to 
-        send to the driver.
-    pulse_shift
-        Phase shift for this pulse, for doing pulse coding.
-    iscombined
-        Boolean, true if there is another pulse with the same combined_pulse_index.
-    combine_total
-        Total number of pulse dictionaries that have the same combined_pulse_index as 
-        this one. (minimum number = 1, itself).
-    combine_index
-        Index of this pulse dictionary in regards to all the other pulse dictionaries that
-        have the same combined_pulse_index. 
+    combined_pulses_metadata
+        This list holds dictionary metadata for all pulses in the sequence. This metadata holds all
+        the info needed to combine pulses if pulses are mixed.
```

Comment:
I think it's helpful to understanding to have the dictionary keys that are used for this parameter listed here and what they mean. What metadata does the combined_pulses_metadata contain? Then units, etc. can also be documented.

Reply:
Actually, is this a member of the class, maybe you mean basic_slice_pulses? That is a member that's not documented